redCOMPONENT-COM/redSHOP
# REDSHOP-1352 #comment Adding Pacsoft Label Creation options

#### To test this PR
- Setup Pacsoft
- Set below options and check
  ![packsoft manually](https://cloud.githubusercontent.com/assets/2002921/3053123/5c04df86-e1a4-11e3-8827-eb6e8610574a.png)
  ![packsoft](https://cloud.githubusercontent.com/assets/2002921/3053130/6e48416a-e1a4-11e3-84ba-691dbb29bb7a.png)
